src/background/EventBus.ts

```diff
@@ -0,0 +1,25 @@
+export class BackgroundEventTarget<T extends any = any> {
+  private listeners: Record<string, Handler<T> | undefined> = {};
+
+  public addEventListener(source: string, callback: Handler<T>) {
+    console.log("adding lsitenet", source);
```

Comment:
Typo "listener"

Reply:
What the hell happened here... I must have been drinking?

Reply:
```suggestion
    console.log("adding listener", source);
```